clouds/aws/aws_roles/aws_roles.py

```diff
@@ -0,0 +1,70 @@
+import botocore.errorfactory
+from aws.aws_clients import IAMClient
+from simple_logger.logger import get_logger
+
+LOGGER = get_logger(name=__name__)
+
+
+class AWSRoles:
+    def __init__(self, client: IAMClient):
+        """
+        Manages AWS roles and role-policies.
+        Args:
+            client (IAMClient): AWS roles managements expects an IAM client entity
+        """
+        self.client = client
```

Comment:
this is not needed
get the client where needed

Reply:
ditto (must be called by client instance such as `client._aws_roles_methods_`)